Donsol/ContentView.swift

```diff
@@ -31,14 +20,45 @@ struct ContentView: View {
 
                 HeaderView(game: game)
                 
-                
                 HStack {
-                    CardButtonView(game: game, isFlipped: $topRightCard_isFlipped, isEnabled: $isEnabled, card: game.deck.drawCard())
-                    CardButtonView(game: game, isFlipped: $topLeftCard_isFlipped, isEnabled: $isEnabled, card: game.deck.cards[0])
+                    if game.room.count > 0 {
+                        let card1 = game.room[0]
+                        CardButtonView(
+                            game: game,
+                            isFlipped: .constant(card1.isFlipped),
```

Comment:
Since this is now just a `Bool`, we need to wrap it with the `.constant` `Binding` type to just pass the value.